TOC-tidb-cloud.md

```diff
@@ -795,7 +795,14 @@
 - Support Plan
   - [Connected Care Overview](/tidb-cloud/connected-care-overview.md)
```

Comment:
Please update the Overview and Details page to point the referenced features to the newly added docs with hyperlinks.